index.html

```diff
@@ -193,28 +231,28 @@ <h3>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod temp
               required
               name="messages" 
               id="" 
-              cols="30" 
-              rows="10">
+              cols="100" 
+              rows="5"
+              >
             </textarea>
           </div>
         </div>
         <button type="submit" class="button-get-started">Start collaboration</button>
```

Comment:
As the previous reviewer requested, please follow the Figma template and fix the width of the submit button.
<img width="947" alt="Screenshot 2023-02-25 at 8 35 24 PM" src="https://user-images.githubusercontent.com/25887253/221371488-f917d7da-b652-4a5f-92df-6cb94b7b6ffe.png">
<img width="794" alt="Screenshot 2023-02-25 at 8 37 34 PM" src="https://user-images.githubusercontent.com/25887253/221371489-675ea302-527e-4fa8-89ac-d2686fc8db18.png">

Reply:
fixes